### ISSUE TITLE
Export image

### ISSUE THREAD

**Issue by Dambond** [chrieke/prettymapp#12]
can you add the possibility to export image in SVG or other file type?

**Comment by chrieke** [comment]
Absolutely! Which other formats do you have in mind? I was thinking svg & georeferenced geotiff might be nice to drop the map image in a map.

SVG export is in fact already [in the code](https://github.com/chrieke/prettymapp/blob/5322d3667b71c22a0b813eeb342743e1f2265617/streamlit-prettymapp/app.py#L188) but commented, due to streamlit not providing a good way to generate the svg string only after the download button is clicked (they are working on that though). Currently it would always run it, which would slow down the app for the other 99% uses.

**Comment by chrieke** [comment]
Thanks for ping @tomaszglinski ! Finally added, will be released in new version very soon.
